src/NuGet.Services.AzureSearch/ICommand.cs

```diff
@@ -3,9 +3,9 @@
 
 using System.Threading.Tasks;
 
-namespace NuGet.Services.AzureSearch.Db2AzureSearch
+namespace NuGet.Services.AzureSearch
 {
-    public interface IDb2AzureSearchCommand
+    public interface ICommand
```

Comment:
this looks job infra related. Is there a reason to use a different model here then we use for other jobs?

Reply:
This is called by a `Job`. It's so the whole thing can be activated by DI.

Reply:
Spoke offline. Will remove this unnecessary interface.